src/api/kotlin/xyz/wagyourtail/unimined/api/minecraft/patch/reindev/FoxLoaderPatcher.kt

```diff
@@ -0,0 +1,32 @@
+package xyz.wagyourtail.unimined.api.minecraft.patch.reindev
+
+import groovy.lang.Closure
+import groovy.lang.DelegatesTo
+import org.gradle.api.artifacts.Dependency
+import xyz.wagyourtail.unimined.api.minecraft.patch.jarmod.JarModPatcher
+
+/**
+ * The class responsible for patching ReIndev for FoxLoader.
+ * @since 1.4.0
+ */
+interface FoxLoaderPatcher : JarModPatcher {
```

Comment:
TODO: FoxLoader mod metadata fields (this loader has no mod meta file, it uses the Java manifest)